src/services/ioSignService.ts

```diff
@@ -57,7 +57,12 @@ const userNotFound =
   "The user associated with this profile could not be found.";
 
 export const getEnvironmentFromHeaders = flow(
```

Comment:
what if our fetch implementation changes?

Reply:
if our fetch implementation changes we will have giant problems all over the project :) the autogenerated client is not fetch-agnostic

Reply:
can unit tests catch it?

Reply:
if there are used other fetch implementation this method will return `prod` (the fallback value), there are no crashes or unhandled behavior